### ISSUE TITLE
Negative input values are not treated (right?)

### ISSUE THREAD

**Issue by LucasF27** [lara-unb/hasomed_rehastim_stimulator#1]
This could lead to unpredictable (and dangerous) behavior.

**Comment by georgebrindeiro-vtex** [comment]
You're right. I'll try to create a few unit tests to simulate this (and other) type(s) of input and attempt to make it safer. Do you have any specific scenario in mind?

**Comment by LucasF27** [comment]
I noticed it when I tested the Rowing system yesterday. I did not have the stimulator pluged in, but the Arduino board accepted negative values (I created an issue there too) and it could send them to the stimulator in case it was all connected. 

As the last safety measure is the stimulator node, we should meake sure it is treated there. 

I recommend stopping the stimulation if it receives a negative value. And possibly an error message.
